iidm/iidm-modification/src/test/java/com/powsybl/iidm/modification/PercentChangeLoadModificationTest.java

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.modification;
+
+import com.powsybl.commons.PowsyblException;
+import com.powsybl.iidm.network.Load;
+import com.powsybl.iidm.network.Network;
+import com.powsybl.iidm.network.test.EurostagTutorialExample1Factory;
+import org.junit.jupiter.api.Test;
+
+import static com.powsybl.iidm.modification.NetworkModificationImpact.*;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+/**
+ * @author Benoît Chiquet {@literal <benoit.chiquet at rte-france.com>}
+ */
+class PercentChangeLoadModificationTest {
+
+    @Test
+    void shouldIncreaseP0() {
+        Network network = EurostagTutorialExample1Factory.create();
+        Load load = network.getLoad("LOAD");
+        assertEquals(600.0, load.getP0());
+        PercentChangeLoadModification modification = new PercentChangeLoadModification("LOAD", 3.51, 0);
+        modification.apply(network);
+        assertEquals(621.06, load.getP0());
```

Comment:
When comparing doubles, it is a good practice to use a delta:

```suggestion
        assertEquals(621.06, load.getP0(), 0.01);
```